subsys/mgmt/mcumgr/grp/img_mgmt/src/zephyr_img_mgmt.c

```diff
@@ -25,12 +25,19 @@ LOG_MODULE_DECLARE(mcumgr_img_grp, CONFIG_MCUMGR_GRP_IMG_LOG_LEVEL);
 #define SLOT1_PARTITION		slot1_partition
 #define SLOT2_PARTITION		slot2_partition
 #define SLOT3_PARTITION		slot3_partition
+#define SLOT4_PARTITION		slot4_partition
+#define SLOT5_PARTITION		slot5_partition
 
 BUILD_ASSERT(CONFIG_MCUMGR_GRP_IMG_UPDATABLE_IMAGE_NUMBER == 1 ||
```

Comment:
Should we demand existence of `slot0_partition` and `slot1_partition` if there is only one updateable image? Or these partitions may not be defined in some cases?

Reply:
Btw. for `CONFIG_MCUMGR_GRP_IMG_UPDATABLE_IMAGE_NUMBER == 3` the check would still not work (slot4 and slot5 existence is not checked).

Seems like `CONFIG_MCUMGR_GRP_IMG_UPDATABLE_IMAGE_NUMBER >= 2` should be `CONFIG_MCUMGR_GRP_IMG_UPDATABLE_IMAGE_NUMBER <= 2`. btw. https://github.com/zephyrproject-rtos/zephyr/pull/59724/commits/7680cce3fc0c705cb6395f7b955d68c2bf89e5e8#r1299914826 seems way simpler and less error prone